### PR TITLE
PRIME-2129 Adjust mandatory prescriber Id's for LPN

### PR DESCRIPTION
**Pull request by hogmann**
1. create an enrollee
2. go to the regulatory screen
3. select college of nurses...
4. select LPN category
5. verify all types don't ask for prescriber ID

### CODE REVIEW

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs [hunk 1]
```diff
@@ -102,6 +103,9 @@ public override IEnumerable<LicenseDetail> SeedData
...
                     new LicenseDetail { Id = 88, LicenseCode = 63, Prefix = "98", Manual = true,  Validate = true,  NamedInImReg = true,  LicensedToProvideCare = false,                                                EffectiveDate = ImRegUpdate1Date, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
 
+                    new LicenseDetail { Id = 89, LicenseCode = 52, Prefix = "L9", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true, EffectiveDate = new(2022, 2, 2, 8, 0, 0, DateTimeKind.Utc), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseDetail { Id = 90, LicenseCode = 55, Prefix = "L9", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true, EffectiveDate = new(2022, 2, 2, 8, 0, 0, DateTimeKind.Utc), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs:107]: Why add new license details when the existing ones were in error? Seems more accurate to update the existing ones

Comment (hogmann) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs]: pushed

**Review by johnnyl6 — APPROVED** [review]
🚀

**Review by AMiKaH — APPROVED** [review]
Looks good!

**Review by james-hollinger — APPROVED** [review]
LGTM

**Pull request merged**